### PR TITLE
Add README on linux max GPU

### PR DESCRIPTION
**Pull request by JinBridger**

### CODE REVIEW

File: README_linux_max.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+export BIGDL_LLM_XMX_DISABLED=1
+export BIGDL_QUANTIZE_KV_CACHE=1
+
+export BIGDL_IMPORT_IPEX=0
+export no_proxy=localhost,127.0.0.1
+
+python startup.py -a
+```
+>[!Note]
+> Please note that `libtcmalloc.so` can be installed by `conda install -c conda-forge -y gperftools=2.10`
```
Comment (Oscilloscope98) [inline README_linux_max.md]: Put it directly in step, not in note box

Comment (Oscilloscope98) [reply README_linux_max.md]: That is, after conda activate

**Comment by Oscilloscope98** [comment]
@shane-huang

Here are the draft setup guide for Intel Data Center GPU Max users :)

**Review by shane-huang — APPROVED** [review]
LGTM.

**Pull request merged**